vulture_os/system/config/views.py

```diff
@@ -107,10 +106,12 @@ def render_form(**kwargs):
         if "internal_tenants" in form.changed_data:
             Cluster.api_request("services.rsyslogd.rsyslog.configure_pstats")
             Cluster.api_request("services.rsyslogd.rsyslog.restart_service")
-        if "portal_cookie_name" in form.changed_data:
+        if "portal_cookie_name" or "public_token" in form.changed_data:
             # Reload Frontends with authenticated Workflows: session checks must be updated with the new cookie's name
             for frontend in Frontend.objects.filter(workflow__authentication__isnull=False).distinct():
-                frontend.reload_conf()
+                for node in frontend.reload_conf():
+                    node.api_request("workflow.workflow.build_conf", workflow.pk)
```

Comment:
`workflow` is undefined!

Reply:
You are right, I fixed it and added vulture's idps reload to it